src/django/requirements.txt

```diff
@@ -1,10 +1,9 @@
 boto3==1.9.92
 dedupe==1.9.4
-django-allauth==0.37.0
 django-amazon-ses==2.0.0
 django-extensions==2.1.4
 django-jsonview==1.2.0
-django-rest-auth==0.9.3
+django-rest-auth[with_social]==0.9.3
```

Comment:
That makes sense. This was done specifically because the `with_social` version brings in `allauth`, so we were able to remove `allauth`. I'll update the commit to indicate that.